weblab/templates/entities/entity_runexperiments.html

```diff
@@ -0,0 +1,75 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+{% load entities %}
+
+{% block title %}Experiment: run specific combinations - {% endblock title %}
+
+{% block body_id %}runexperiment{% endblock %}
+
+{% block content %}
+  <h1 id="entityname">Run experiments using <span>{{ entity.name }}</span></h1>
+    <!-- SK TO DO: make these work -->
+     <p>
+         <input class="overrideCheckBox" type="checkbox" value="false"/>
+          Force overwrite of existing experiments
```

Comment:
Actually, this could probably be worded better - it won't actually overwrite, but will create a new version. Perhaps 'Create new versions of existing experiments (if unchecked, existing combinations will be skipped)'?

Reply:
That looked okay; so have changed it as above